### ISSUE TITLE
Creating a prepare_glint.py file

### ISSUE THREAD

**Issue by nikky4D** [layumi/Person_reID_baseline_pytorch#319]
I want to train this on glint360K. But I'm not sure how to create the folder directory needed. Glink360K has a folder with 100K folders, each folder is for one id. So in train_folder, we have folder id_0, folder id_1, ..., folder id_99999. In each id_[n] folder, there are images of the same person. From following prepare_market.py, I don't see how to create the various folders required. Do you have any advice on how to create my own .py file for dataloading for DG-Net?

**Comment by layumi** [comment]
Hi @nikky4D 
"Glink360K has a folder with 100K folders, each folder is for one id. So in train_folder, we have folder id_0, folder id_1, ..., folder id_99999. "
You do not need to further prepare actually. You only need to change the training set path. 

You need to sample one image from every folder to make a validation set.

**Comment by nikky4D** [comment]
Thanks. I was thinking along those lines for train folder. But I was not sure what to do about the query, and gallery? Should I just take a set of ids from the main folder as the gallery, and some images from the gallery as the query?

**Comment by layumi** [comment]
@nikky4D 
You need to leave some unseen ids for the test (query and gallery). 
If you have 1000 ids, you may leave 100 ids as the test set. 
The query folder can be extracted from these 100 ids. (One id one camera we select one image as query) 
Then you can put the rest unselected images into the gallery folder.
